app/assets/stylesheets/modules/home.scss

```diff
@@ -43,3 +43,12 @@
 #all_users.well {
   background-color: $white;
 }
+
+.nav > li > a.exhibit-inline {
```

Comment:
@tingulfsen just FYI what they meant was they wanted `.exhibit-inline` instead of `a.exhibit-inline` (i.e. drop the qualifying `a` element).  I ended up tweaking this a bit to go w/ another approach so that the icon was part of the image, but just thought I would mention it in case you see this crop up again.